src/main.rs

```diff
@@ -1,40 +1,134 @@
 extern crate clap;
 extern crate ignore;
+extern crate regex;
+extern crate difference;
+extern crate term;
 
 use clap::{App, Arg};
 use ignore::Walk;
+use std::path::Path;
+use std::fs::File;
+use std::io::prelude::*;
+use regex::Regex;
+use difference::{Changeset, Difference};
+
+#[allow(unused_must_use)]
+fn term_show_diff(before: &str, after: &str) {
+   // Compare both texts, the third parameter defines the split level.
+    let Changeset { diffs, .. } = Changeset::new(before, after, "\n");
+
+    let mut t = term::stdout().unwrap();
+
+    for i in 0..diffs.len() {
```

Comment:
for diff in &diffs ?

Reply:
I'll try. (copy pasted also). it has to be rewritten.

Reply:
fixed